docs/2-Interviews/preparing_for_interviews.md

```diff
@@ -12,22 +12,64 @@ Be prepared to:
 - talk about and work towards writing an algorithm to solve a problem, during the interview
 - you can also expect a “tell me about yourself” question (so be prepared!) and some time where they tell you about their company
 
+### What material should I know?
+
+This is a difficult question to answer. The more you know, the fewer the number of weak points that could trip you up. Knowing more things is a good thing in general, you shouldn't learn things just for interviews.
+
+That being said, you have limited time to learn things so you probably want the biggest bang for the buck.
+
+Things that interviewers will expect you to know:
```

Comment:
I'm thinking I could write up a few different tiers of what companies might expect? I always feel a bit icky drawing arbitrary lines but it can be useful...

(One thing that I was going to write later is "understanding what an internship mean" and talk about how the main goal is to learn - therefore lots companies are totally ok with students coming in not knowing much at all).